pac4j-core/src/main/java/org/pac4j/core/authorization/authorizer/OrAuthorizer.java

```diff
@@ -0,0 +1,36 @@
+package org.pac4j.core.authorization.authorizer;
+
+import org.pac4j.core.context.WebContext;
+import org.pac4j.core.profile.CommonProfile;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+/**
+ * The disjunction of authorizers.
+ * @param <U> Type of profile
+ *
+ * @author Sergey Morgunov
+ * @since 3.4.0
+ */
+public class OrAuthorizer<U extends CommonProfile> implements Authorizer<U> {
```

Comment:
Can you add a test class for this `OrAuthorizer `?

Reply:
Added.